packages/windbg.vm/tools/chocolateyinstall.ps1

```diff
@@ -5,16 +5,23 @@ try {
     $toolName = 'WinDbg'
     $category = 'Debuggers'
 
-    # It seems WinDbg is now distributed as an .appinstaller and we need to install it using Add-AppxPackage
-    Add-AppxPackage -AppInstallerFile 'https://windbg.download.prss.microsoft.com/dbazure/prod/1-0-0/windbg.appinstaller'
+    $exeUrl = "https://windbg.download.prss.microsoft.com/dbazure/prod/1-2402-24001-0/windbg.msixbundle"
+    $exeSha256 = "e941076cb4d7912d32a22ea87ad2693c01fa465227b4d1ead588283518de428f"
 
-    $shortcutDir = Join-Path ${Env:TOOL_LIST_DIR} $category
-    $shortcut = Join-Path $shortcutDir "$toolName.lnk"
-    $executableCmd  = Join-Path ${Env:WinDir} "system32\cmd.exe"
-    # Use `start` to close the open console
-    $executableArgs = "/C start WinDbgX.exe"
-    $executableDir  = Join-Path ${Env:UserProfile} "Desktop"
-    Install-ChocolateyShortcut -shortcutFilePath $shortcut -targetPath $executableCmd -Arguments $executableArgs -WorkingDirectory $executableDir -RunAsAdmin
+    $packageArgs = @{
+        packageName   = ${Env:ChocolateyPackageName}
+        url           = $exeUrl
+        checksum      = $exeSha256
+        checksumType  = "sha256"
+        fileFullPath  = Join-Path ${Env:TEMP} "$toolName.msixbundle"
+    }
+    Get-ChocolateyWebFile @packageArgs
+    Add-AppxPackage -Path $packageArgs.fileFullPath
+
+    $installDir = (Get-AppxPackage -Name "Microsoft.$toolName").InstallLocation
+    $iconLocation  = Join-Path $installDir "DbgX.Shell.exe" -Resolve
+    $executablePath = "$(where.exe WinDbgX.exe)"
```

Comment:
it seems like the windows-2019 run can't find `WinDbgX.exe` which likely mean it hasn't been installed... (likely it was also not working before, but we were not checking). The GH runner is using Windows `10.0.17763`, locally I am using `10.0.19045` for testing.

Searching online about issues installing WinDbg, I see information about conflicts with the Windows Software Development Kit. For example: https://superuser.com/questions/1524982/windbg-repeatedly-downloads-but-doesnt-install. Maybe requiring `windows-sdk-10.0` as dependency fixes the issue. :thinking: 

@mandiant/vms any other ideas how to fix this?

Reply:
Unfortunately, I won't be able to test it myself as I don't have any Windows 10 VM and can't get one easily to test for now.

Reply:
I have Windows 10, but newer than the one in GH actions and it works for me. I am also not sure how to get an older Windows 10 for local testing. We can use GH for testing by pushing new changes to this PR (or running the test in a fork to avoid the noise in the PR).

@mandiant/vms any ideas to fix or test this problem?

Reply:
> Searching online about issues installing WinDbg, I see information about conflicts with the Windows Software Development Kit. For example: https://superuser.com/questions/1524982/windbg-repeatedly-downloads-but-doesnt-install. Maybe requiring `windows-sdk-10.0` as dependency fixes the issue. 🤔

I tried to add `windows-sdk-10.0` as dependency but it still failed.

Reply:
If we do not figure out what the issue is, we can keep running it as a console app with:
```
VM-Install-Shortcut -toolName $toolName -category $category -executablePath "start $toolName.exe" -consoleApp $true -runAsAdmin -iconLocation $iconLocation
```

Reply:
I have tested this, however I get `'"start WinDbgX.exe"' is not recognized as an internal or external command, operable program or batch file.` because of the double quote added inside `VM-Install-Shortcut`.

`VM-Install-Shortcut -toolName $toolName -category $category -executablePath "start WinDbgX.exe" -consoleApp $true -RunAsAdmin -iconLocation $iconLocation`

Reply:
@Ana06, I have tried another idea by catching the exception, it seems to work !

Reply:
I do not like that we use `$executablePath = Join-Path $installDir "DbgX.Shell.exe"` as executing `DbgX.Shell.exe` does not work for me, so potentially this is broken for others too. I think without the `start` it should work and be equivalent to the code we had before:

VM-Install-Shortcut -toolName $toolName -category $category -executablePath "$toolName.exe" -consoleApp $true -runAsAdmin -iconLocation $iconLocation

Reply:
I have noticed I can't even accces `C:\Program Files\WindowsApps\` where `DbgX.Shell.exe` is, likely why I can't execute it
![image](https://github.com/mandiant/VM-Packages/assets/16052290/a2c7a47b-ca02-4093-be73-cabc65d984f4)

![image](https://github.com/mandiant/VM-Packages/assets/16052290/53cce547-9f46-4e36-8e18-e9b64cd04722)

Reply:
> I have noticed I can't even accces `C:\Program Files\WindowsApps\` where `DbgX.Shell.exe` is, likely why I can't execute it


I don't think it's related, at least not directly. Since you should be able to access`C:\Program Files\WindowsApps\Microsoft.WinDbg_1.2402.24001.0_x64__8wekyb3d8bbwe`.  in my installation (which works with `DbgX.Shell.exe`) I don't have access to WinDbg either.


I don't think it's related at least directly. Since you should be able to access `C:\Program Files\WindowsApps\Microsoft.WinDbg_1.2402.24001.0_x64__8wekyb3d8bbwe\`. Besides, in my setup (which works with `DbgX.Shell.exe`) I don't have access to `C:\Program Files\WindowsApps\` either.

Reply:
> I do not like that we use `$executablePath = Join-Path $installDir "DbgX.Shell.exe"` as executing `DbgX.Shell.exe` does not work for me, so potentially this is broken for others too. 

I only use it when `which` fails, so only for older versions of Windows 10, it seems. According to the current  `windbg.vm` code, it shouldn't work for these older versions of Windows either because `WinDbgX` is hard-coded without `VM-Assert-Path`.

> I think without the `start` it should work and be equivalent to the code we had before:
> 
> `VM-Install-Shortcut -toolName $toolName -category $category -executablePath "$toolName.exe" -consoleApp $true -runAsAdmin -iconLocation $iconLocation `

If I do this, it will still not work for older versions of Windows 10, I think my solution might give it a chance to actually work.